docs/src/guides/run-genbank-ingest-on-terra.rst

```diff
@@ -0,0 +1,69 @@
+*********************************************
+Ingest SARS-CoV-2 data from GenBank on Terra
+*********************************************
+
+We have provided two pipelines for importing data into Terra:
+
+* **GenBank Ingest - pull a public dataset and send them through our preprocessing scripts.**
+* GISAID Ingest - pull a private dataset if a user has their own API endpoint, username, and password.
+
+The pipelines were mainly motivated to provide access to our data pre-processing scripts. Currently, these are focused on pulling datasets for the ncov workflow and is roughly diagrammed below:
+
+.. image:: ../images/terra-ingest.png
+
+This guide describes the **GenBank Ingest**. Importing the workflow and setting up variables should be a one-time process. The workflow can then be run multiple times to pull in new data.
+
+As of October 29 2022, this took `37 hours to finish running with default parameters, and cost ~$150 USD <https://github.com/j23414/wdl_pathogen_build/blob/main/data/benchmarks.tsv>`_.
+
+Import the GenBank ingest wdl workflow from Dockstore
+======================================================
+
+1. `Set up a Terra account <https://terra.bio/>`_.
+2. Navigate to Dockstore: `nextstrain/ncov/genbank_ingest <https://dockstore.org/workflows/github.com/nextstrain/ncov/genbank_ingest:master?tab=info>`_
+3. At the top right corner, under **Launch with**, click on **Terra**. You may be prompted to log in.
+4. Provide a **Workflow Name** (e.g. ``genbank_ingest`` ).
+5. Select a **Destination Workspace** from the dropdown menu.
+6. Click **IMPORT**.
+7. In your workspace, click on the **WORKFLOWS** tab and verify that the imported workflow is showing a card.
+
+Connect any workspace variables to the wdl ingest workflow
+===========================================================
+  
+1. Navigate back to the **Workflow** tab, and click on the workflow imported to your workspace.
+2. Click on the radio button **Run workflow(s) with inputs defined by data table**.
+3. Under **Step 1**:
+
+  1. Select root entity type as **ncov_examples** from the drop down menu.
```

Comment:
> I'd imagine users would want to run ingest first, followed by the workflow that produces builds.

We need to clarify this. @huddlej? I'm expecting users to first encounter "run-analysis-on-terra" similar to our SARS-CoV-2 Workflow documentation, where they upload an example metadata and sequences files.

* https://docs.nextstrain.org/projects/ncov/en/latest/tutorial/example-data.html

After which, the user learns how to pull and include their own context sequences (equivalent to `run-XXX-ingest-on-terra`:

* https://docs.nextstrain.org/projects/ncov/en/latest/tutorial/genomic-surveillance.html

But this is a good point, I will reword the beginning of this tutorial to explicitly recommend tutorial order. I think we (or maybe just me) have gotten confused because this tutorial was also trying to address a very specific need (where that individual group would need `ingest`, then `ncov`).